src/test/py/bazel/bzlmod/bazel_module_test.py

```diff
@@ -23,6 +23,17 @@
 from src.test.py.bazel.bzlmod.test_utils import BazelRegistry
 from src.test.py.bazel.bzlmod.test_utils import scratchFile
 
+_OVERRIDE = """
```

Comment:
Is this still needed after rules_cc has your changes?

Reply:
nope, just need that update